build/dockerfiles/rhel.cache_projects.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
I'm not sure this script belongs in this repo -- it's not referenced by the default rhel.Dockerfile and is *only* used in the downstream build.

Reply:
but what does it hurt, being upstream and colocated with rhel.Dockerfile?

Reply:
It's strange to include a script that's never used, but is required for downstream -- someone could modify this script (or remove it, after noticing that it's unused) and break downstream.

Reply:
ok, sure but the rhel.Dockerfile is also never used in upstream but required in down. 

Including it in up serves the purpose of reminding people to think about downstream when making changes in up.

Reply:
Fair enough -- it's your call. However, rhel.Dockerfile *is* used, in the sense that it's a build option in this repository; the script is not standalone and is called nowhere. It's not even clear in the context of this repo where it should be called (since in midstream we call it during the builder phase instead of the `offline-builder` phase).